src/pages/Login/Login.scss

```
@@ -0,0 +1,74 @@
+.login {
+  width: 100%;
+  height: 100%;
+  font-family: 'SUIT', sans-serif;
+  // background-color: red;
+  text-align: center;
+  word-break: keep-all; // 문장을 문단으로 구분
+  .loginWelcome {
+    width: 400px;
+    height: 67px;
+    margin: 0 auto;
+    // background-color: blue;
+    font-size: 24px;
+    p {
+      // Nav와 안내문구와 사이 공백을 넣어줌
+      display: block; // display속성을 inline이 아닌 block으로 표시
+      margin-block-start: 1em;
+      margin-block-end: 1em;
+      margin-inline-start: 0px;
+      margin-inline-end: 0px;
+    }
+  }
+  .loginMain {
+    width: 400px;
+    height: 421px;
+    margin: 0 auto;
+    // background-color: yellow;
+    font-size: 14px;
+    .loginMainInputId {
```

```suggestion
  .loginMain {
    width: 400px;
    height: 421px;
    margin: 0 auto;
    // background-color: yellow;
    font-size: 14px;
    
    .loginMainInputId {
```
불필요한 주석은 제거해주시고, 스타일 간 가독성을 위해 한칸씩 띄워주세요!

---

반영하였습니다! 감사합니다!